### PR TITLE
Feature/M2-2016 Builder - Flanker - General Settings (Buttons, Fixation Screen, Stimulus Screen)

### PR DESCRIPTION
**Pull request by IhorNesterenko99**

### CODE REVIEW

File: src/modules/Builder/components/ToggleItemContainer/ToggleItemContainer.styles.ts [hunk 1]
```diff
@@ -23,7 +23,10 @@ export const StyledItemOption = styled(StyledItemOptionContainer, shouldForwardP
 `;
 
 export const StylesTitleWrapper = styled(StyledFlexTopCenter, shouldForwardProp)`
-  ${({ open }: { open?: boolean }) => `
-		height: ${open ? '5.6rem' : 'inherited'}; 
+  justify-content: space-between;
+
+  ${({ open, uiType }: { open?: boolean; uiType: ToggleContainerUiType }) => `
+		height: ${open ? '5.6rem' : 'inherit'};
```
Comment (kdanilenka32) [inline src/modules/Builder/components/ToggleItemContainer/ToggleItemContainer.styles.ts:29]: smth with spaces?

File: src/modules/Builder/features/PerformanceTasks/Flanker/GeneralSettings/ButtonsScreen/ButtonsContent/ButtonsContent.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <Box sx={{ width: '23.1rem', mb: theme.spacing(2.4) }}>
+        <ToggleButtonGroup
+          toggleButtons={buttonsQuantity}
+          activeButton={activeButton}
+          setActiveButton={handleActiveBtnChange}
+        />
+      </Box>
+      <StyledFlexSpaceBetween>
+        {buttons?.length &&
+          buttons.map((button, index) => (
```
Comment (kdanilenka32) [inline src/modules/Builder/features/PerformanceTasks/Flanker/GeneralSettings/ButtonsScreen/ButtonsContent/ButtonsContent.tsx:53]: buttons?.map

**Review by muhurov — APPROVED** [review]

File: src/modules/Builder/features/PerformanceTasks/Flanker/GeneralSettings/StimulusScreen/StimulusContent/StimulusContent.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        </StyledHeader>
+      ) : (
+        <StyledInfoSection>
+          <StyledBodyLarge color={variables.palette.outline}>
+            {t('flankerStimulus.addText')}
+          </StyledBodyLarge>
+        </StyledInfoSection>
+      )}
+      {!!stimulusTrials?.length &&
+        stimulusTrials.map((trial, index) => (
```
Comment (kdanilenka32) [inline src/modules/Builder/features/PerformanceTasks/Flanker/GeneralSettings/StimulusScreen/StimulusContent/StimulusContent.tsx:66]: stimulusTrials?.map

File: src/modules/Builder/components/ToggleItemContainer/ToggleItemContainer.styles.ts (hunk 1 above)
Comment (IhorNesterenko99) [reply src/modules/Builder/components/ToggleItemContainer/ToggleItemContainer.styles.ts:29]: In IDE everything looks good:
image

**Review by kdanilenka32 — APPROVED** [review]

**Pull request merged**